ray-operator/test/e2e/rayservice_ha_test.go

```diff
@@ -5,83 +5,107 @@ import (
-			WithSpec(rayv1ac.RayClusterSpec().
-				WithRayVersion(GetRayVersion()).
-				WithHeadGroupSpec(rayv1ac.HeadGroupSpec().
-					WithRayStartParams(map[string]string{"dashboard-host": "0.0.0.0"}).
-					WithTemplate(apply(headPodTemplateApplyConfiguration(), mountConfigMap[corev1ac.PodTemplateSpecApplyConfiguration](scripts, "/home/ray/test_scripts")))))
-		locustCluster, err := test.Client().Ray().RayV1().RayClusters(namespace.Name).Apply(test.Ctx(), locustClusterAC, TestApplyOptions)
-		g.Expect(err).NotTo(HaveOccurred())
-		test.T().Logf("Created Locust RayCluster %s/%s successfully", locustCluster.Namespace, locustCluster.Name)
-
-		// Wait for RayCluster to become ready and verify the number of available worker replicas.
-		g.Eventually(RayCluster(test, locustCluster.Namespace, locustCluster.Name), TestTimeoutMedium).
-			Should(WithTransform(RayClusterState, Equal(rayv1.Ready)))
-		g.Expect(GetRayCluster(test, locustCluster.Namespace, locustCluster.Name)).To(WithTransform(RayClusterDesiredWorkerReplicas, Equal(int32(0))))
-
-		headPod, err := GetHeadPod(test, locustCluster)
-		g.Expect(err).NotTo(HaveOccurred())
-		test.T().Logf("Found head pod %s/%s", headPod.Namespace, headPod.Name)
-
-		// Install Locust in the head Pod
-		ExecPodCmd(test, headPod, common.RayHeadContainer, []string{"pip", "install", "locust"})
-
-		// Run Locust test
-		ExecPodCmd(test, headPod, common.RayHeadContainer, []string{
-			"python", "/home/ray/test_scripts/locust_runner.py", "-f", "/home/ray/test_scripts/locustfile.py", "--host", "http://static-raysvc-serve-svc:8000",
-		})
+	test.T().Logf("Created RayService %s/%s successfully", rayService.Namespace, rayService.Name)
+
+	test.T().Logf("Waiting for RayService %s/%s to running", rayService.Namespace, rayService.Name)
+	g.Eventually(RayService(test, rayService.Namespace, rayService.Name), TestTimeoutMedium).
+		Should(WithTransform(RayServiceStatus, Equal(rayv1.Running)))
+
+	// Create Locust RayCluster
+	KubectlApplyYAML(test, locustYamlFile, namespace.Name)
+	locustCluster, err := GetRayCluster(test, namespace.Name, "locust-cluster")
+	g.Expect(err).NotTo(HaveOccurred())
+	test.T().Logf("Created Locust RayCluster %s/%s successfully", locustCluster.Namespace, locustCluster.Name)
+
+	g.Eventually(RayCluster(test, locustCluster.Namespace, locustCluster.Name), TestTimeoutMedium).
+		Should(WithTransform(RayClusterState, Equal(rayv1.Ready)))
+	g.Expect(GetRayCluster(test, locustCluster.Namespace, locustCluster.Name)).To(WithTransform(RayClusterDesiredWorkerReplicas, Equal(int32(0))))
+
+	headPod, err := GetHeadPod(test, locustCluster)
+	g.Expect(err).NotTo(HaveOccurred())
+	test.T().Logf("Found head pod %s/%s", headPod.Namespace, headPod.Name)
+
+	// Install Locust in the head Pod
+	ExecPodCmd(test, headPod, common.RayHeadContainer, []string{"pip", "install", "locust"})
+
+	// Run Locust test
+	ExecPodCmd(test, headPod, common.RayHeadContainer, []string{
```

Comment:
split `locust_runner` into two parts:

* 10 users -> 120 users
  * Check the number of Pods
* 120 users -> 10 users
  * Check the number of Pods

Reply:
Implement the first part is enough. So I only implement the first part.